flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveTableConfig.java

```diff
@@ -19,11 +19,15 @@
 package org.apache.flink.table.catalog.hive;
 
 /**
- * Configs for Flink tables stored in Hive metastore.
+ * Configs for tables in Hive metastore.
  */
 public class HiveTableConfig {
 
-	// Description of the Flink table
+	// -------------------
```

Comment:
Do the three lines (26-28) provide any additional info above the class name here?

Reply:
removed